SCOPE.md

```diff
@@ -0,0 +1,16 @@
+# Scope
+
+Specifies procedures for constructing and conveying artifact trees, directed acyclic graphs (DAGs), and other related 
```

Comment:
Suggestion: drop "artifact tree" throughout, since we're using DAG generally in the spec.

Reply:
I've dropped artifact trees.

Thoughts on 'artifact dependency graphs (ADGs)' as a moniker?  I'm open to considering 'artifact dependency directed acyclic graphs (ADDAGs or 'artifact dependency DAGs).  I'm mostly interested in thoughts and opinions :)